searx/autocomplete.py

```diff
@@ -164,6 +164,21 @@ def qihu360search(query, _lang):
     return results
 
 
+def quark(query, _lang):
+    # Quark search autocompleter
+    url = f"https://sugs.m.sm.cn/web?{urlencode({'q': query})}"
+    response = get(url)
+
+    results = []
+
+    if response.ok:
+        data = response.json()
+        if 'r' in data:
+            for item in data['r']:
```

Comment:
```suggestion
            for item in data.get('r', []):
```
This makes the line above redundant.

Reply:
Done